products.js

```diff
@@ -18,20 +18,37 @@ const {dom, props, out, rule, ruleset, score, type} = require('fathom-web');
 const {Annealer} = require('fathom-web/optimizers');
 const {staticDom} = require('fathom-web/utils');
 const tuningRoutines = {
-                        // 'title' : {'routine': tunedTitleFnodes, 'coeffs': []},
+                        'title' : {'routine': tunedTitleFnodes, 'coeffs': []},
                         'price' : {'routine': tunedPriceFnodes, 'coeffs':  [ 4.4, 3, 100, 2, 5, 2.6, 160, 2.6, 0.4, 0.2, 0.5, 0.2, 0.5, 4.4, 1.6, 0.8, 0.2, 0.05, 2.6, 0.65, 440]},
-                        // 'image' : {'routine': tunedImageFnodes, 'coeffs': [1.9, 3.0, 420.0, 500.0, 0.05, 800.0, 1300.0, 0.7, 0.2, 0.5, 0.1, 0.1, 1.3]}
+                        'image' : {'routine': tunedImageFnodes, 'coeffs': [1.9, 3.0, 420.0, 500.0, 0.05, 800.0, 1300.0, 0.7, 0.2, 0.5, 0.1, 0.1, 1.3]}
                         };
-const VIEWPORT_HEIGHT = 960;
 const VIEWPORT_WIDTH = 1680;
+const VIEWPORT_HEIGHT = 960;
+
+/*
+ * Remove dollar sign, strip whitespace, strip words (anything not numeric or a price symbol), and remove trailing zeros
+ */
+function formatPrice(priceString){
+  priceString = priceString.replace('$', '').replace(/([\s]|[^0-9$.-])/g, '').replace(/(\.[0-9]*?)0+$/, '');
+  if (priceString.includes('$')){
```

Comment:
You use this pattern in a few places: includes() and then a substr(). See if there's already a string method for this. If not, here's a great opportunity to make one. It'll make your code clearer and probably boost performance, since you won't have to linear-scan through the string twice.